app/src/test/java/com/stevesoltys/seedvault/restore/install/ApkBackupRestoreTest.kt

```diff
@@ -133,7 +133,9 @@ internal class ApkBackupRestoreTest : TransportTest() {
         every { strictContext.cacheDir } returns tmpFile
         every { crypto.getNameForApk(salt, packageName, "") } returns name
         coEvery { storagePlugin.getInputStream(token, name) } returns inputStream
-        every { pm.getPackageArchiveInfo(capture(apkPath), any()) } returns packageInfo
+        every {
+            pm.getPackageArchiveInfo(capture(apkPath), PackageManager.PackageInfoFlags.of(any()))
+        } returns packageInfo
```

Comment:
How does it work to switch the tests to the new method, but not the actual code that is being tested?